extendeddatasquare_test.go

```diff
@@ -1,37 +1,75 @@
 package rsmt2d
 
 import (
+	"bytes"
 	"crypto/rand"
 	"encoding/json"
 	"fmt"
 	"reflect"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	zero     = bytes.Repeat([]byte{0}, 64)
+	one      = bytes.Repeat([]byte{1}, 64)
+	two      = bytes.Repeat([]byte{2}, 64)
+	three    = bytes.Repeat([]byte{3}, 64)
+	four     = bytes.Repeat([]byte{4}, 64)
+	five     = bytes.Repeat([]byte{5}, 64)
+	eight    = bytes.Repeat([]byte{8}, 64)
+	eleven   = bytes.Repeat([]byte{11}, 64)
+	thirteen = bytes.Repeat([]byte{13}, 64)
+	fifteen  = bytes.Repeat([]byte{15}, 64)
```

Comment:
[non-blocking]

perhaps we could add "s" to each of these to indicate that they are slices of that byte? 

also perhaps add a quick doc to remind the reader that we have to use slices that are multiples of 64

Reply:
Good idea. I originally had "ones", "twos" but then renamed because I had 

```go
fs  = bytes.Repeat([]byte{0xf}, 64)
```

which felt wrong. Renaming helped.